File: Desarrollo_Movil/AndroidStudio/OnClickListener/app/src/main/java/com/example/onclicklistener/MainActivity.java
package com.example.onclicklistener;

import androidx.appcompat.app.AppCompatActivity;
import android.app.*;
import android.os.*;
import android.view.View;
import android.view.View.*; //OnClickListener //1
import android.widget.*;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {
    EditText jet1;
    Button jbn1;

    @Override
    protected void onCreate(Bundle b) {
        super.onCreate(b);
        setContentView(R.layout.activity_main);
        jet1 = (EditText) findViewById(R.id.xet1);
        jbn1 = (Button) findViewById(R.id.xbn1);
        jbn1.setOnClickListener((OnClickListener) this); //3
    }

    public void onClick(View v)
    {
        int n = Integer.parseInt(jet1.getText()+"");
    }
}

File: Desarrollo_Movil/AndroidStudio/KotlinPDF2/app/src/main/java/com/example/kotlinpdf2/MainActivity.kt
package com.example.kotlinpdf2

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btnScan = findViewById<Button>(R.id.btnVerif)
        val input = findViewById<EditText>(R.id.input)
        var b: String? = "NULO"
        btnScan.setOnClickListener {
            if (input.text.toString() == ""){
                b = null
                Toast.makeText(this,"El valor es nulo pero muestra: " + b, Toast.LENGTH_LONG).show()
            }
            else{
                b = input.text.toString()
                Toast.makeText(this,"El valor no es nulo pero muestra: " + b, Toast.LENGTH_LONG).show()
            }
        }
    }
}
